recipes/pycatch22/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "pycatch22" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycatch22-{{ version }}.tar.gz
+  sha256: 162bd9774a326993c564120221e779fb6e578139e9fb73ed6066a3137e61c3ad
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - pycatch22
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/pycatch22/
+  summary: 22 CAnonical Time-series Features
+  license: GPL-3.0
```

The proper identifier:
```suggestion
  license: GPL-3.0-only
```